massgov/mayflower
# React/DP-10868: A11y TabContainer support.

## Description
This adds a11y support for the Tab and TabContainer components. This is loosely based on https://github.com/massgov/mayflower/pull/353.

## Related Issue / Ticket

- [JIRA issue](https://jira.mass.gov/browse/DP-10868)

## Steps to Test
1. Pull the branch, start up storybook. Browse to organisms -> TabContainer and verify the functionality works as needed.

## Screenshots
![tabcontainer-with-keyboard](https://user-images.githubusercontent.com/5257660/53750937-1b360200-3e79-11e9-9b6e-dcbe9e902b5a.gif)

#### Impacted Areas in Application

* TabContainer
* Tab
